### PR TITLE
RabbitMQ - GH-43 - Support Fix For Routing Key

### PR DESCRIPTION
**Pull request by garyrussell**
See spring-cloud/spring-cloud-stream-binder-rabbit#43

### CODE REVIEW

File: spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/PartitionCapableBinderTests.java [hunk 1]
```diff
@@ -194,8 +194,7 @@ public void testPartitionedModuleSpEL() throws Exception {
 		Binding<MessageChannel> outputBinding = binder.bindProducer("part.0", output, producerProperties);
 		try {
 			Object endpoint = extractEndpoint(outputBinding);
-			assertThat(getEndpointRouting(endpoint))
-					.contains(getExpectedRoutingBaseDestination("part.0", "test") + "-' + headers['partition']");
+			checkRkExpression(endpoint);
```
Comment (mbogoevici) [inline spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/PartitionCapableBinderTests.java]: Do we want to use this name or something more expressive?

**Comment by mbogoevici** [comment]
Merged in master via https://github.com/spring-cloud/spring-cloud-stream/commit/51a7f8bf66633189155fa0cb6e2907b9a8288313

**Comment by artembilan** [comment]
> Merged in master via 51a7f8b

So, let's close then ! 😄

**Comment by mbogoevici** [comment]
@artembilan I was waiting for a confirmation whether we want to backport or not.

**Comment by mbogoevici** [comment]
Cherry-picked as https://github.com/spring-cloud/spring-cloud-stream/commit/8056473892c75c37b3d09a58529352bb45fe5dbd

**Pull request closed without merging**